docs/content/documentation/content/section.md

```diff
@@ -164,6 +165,13 @@ to newest (at the bottom).
 
 `reverse` has no effect on `page.later`/`page.earlier` or `page.heavier`/`page.lighter`.
 
+However this will not work correctly if the pages are paginated, because it
+will only apply to the pagers and not to all pages. For example if you have
+pages `[1,2,3,4]` and a `paginate_by=2` they will be paginated as
+`[[2,1],[4,3]]` instead of `[[4,3],[2,1]]` as you might expect. To achive this
```

Comment:
achieve

Reply:
does that mean you want this part removed?

Reply:
no, just the typo